chat-api/v2/samples/rich_media_template_with_text.md

```diff
@@ -0,0 +1,73 @@
+# Rich Media Template with text
+
+This is an example request for sending a text together with a rich media template
```

Comment:
Add a trailing period after the sentence (watch it in the following places as well)